### ISSUE TITLE
Unable to create new root user using official container

### ISSUE THREAD

**Issue by JustonDavies** [tinode/chat#874]
### Subject of the issue
When running the official docker image (`docker pull tinode/tinode-mysql:0.22.8`) with a local MySQL database I am unable to add a root user with `sudo docker exec ${CONTAINER_NAME} ./init-db --config=/tinode.conf --add_root=root:root123`

#### Server-side
- [ ] web.tinode.co, api.tinode.co
- [ ] sandbox.tinode.co
- [X] Your own setup:
  * platform (Windows, Mac, Linux etc) - Debian Linux 11(6.1.0-0.deb11.7-amd64)
  * version of Tinode server, e.g. `0.15.2-rc3` - v0.22.8
  * database backend - MySQL 8.0.33
  * cluster or standalone - Standalone
  * other details - podman/docker 3.0.1
  
```
CONTAINER ID  IMAGE                                 COMMAND  CREATED      STATUS          PORTS                                             NAMES
33c7b157aadb  docker.io/library/mysql:8             mysqld   3 hours ago  Up 3 hours ago  0.0.0.0:3306->3306/tcp                            tinode_mysql
75446ee88c6a  docker.io/tinode/tinode-mysql:0.22.8           3 hours ago  Up 3 hours ago  0.0.0.0:6060->6060/tcp, 0.0.0.0:16060->16060/tcp  tinode_chat
```

#### Client-side
- [ ] TinodeWeb/tinodejs: javascript client
  * Browser make and version.
  * IMPORTANT! Use `index-dev.html` to reproduce the problem, not `index.html`.
- [ ] Tindroid: Android app
  * Android API level (e.g. 25).
  * Emulator or hardware, if hardware describe it.
- [ ] Tinodios: iOS app
  * iOS version
  * Simulator or hardware, if hardware describe it.
- [ ] tn-cli
  * Python version
- [ ] Chatbot
  * Python version
- Version of the client, e.g. `0.15.1`
- [ ] Your own client. Describe it:
  * Transport (gRPC, websocket, long polling)
  * Programming language.
  * gRPC version, if applicable.
- [X] initd-db

### Steps to reproduce
0. Create docker network with `sudo docker network create tinode_chat_network`
1. Setup and run mysql:8 docker container with ` sudo docker run --name "tinode_mysql" --network tinode_chat_network --env MYSQL_ROOT_PASSWORD="secure_root_password" --publish 3306:3306 --detach mysql:8`
2. Create and modify a default configuration file for tinode in `./tinode.conf`
3. Modify line 272 for the MySQL DSN to read `"dsn": "root:secure_root_password@tcp(tinode_mysql)/tinode_database?parseTime=true"` 
4. Run the official tinode container via `sudo docker run --name "tinode_chat" --network tinode_chat_network --env ./tinode.conf=/tinode.conf --publish 6060:6060 --publish 16060:16060 --detach --rm tinode/tinode-mysql:0.22.8`
5. Wait for the server to start
6. Attempt to create and add a new root user through the documented process via `sudo docker exec tinode_chat ./init-db --config=/tinode.conf --add_root=root:root123`

### Expected behaviour
A new user should appear in the MySQL `users` table with an associated `auth` record that has and `authlvl` of 30

### Actual behaviour
A new user appeared in the `users` table, but no new records appeared in the `auth` table. The output of the command is as follows:
```
2023/06/29 21:28:59 Database adapter: 'mysql'; version: 113
2023/06/29 21:28:59 Database exists, version is correct.
2023/06/29 21:28:59 Failed to create ROOT user: not found
```

### Server-side log
The server generated no additional logfile lines.

### Client-side log
```
2023/06/29 21:28:59 Database adapter: 'mysql'; version: 113
2023/06/29 21:28:59 Database exists, version is correct.
2023/06/29 21:28:59 Failed to create ROOT user: not found
```

**Comment by aforge** [comment]
https://github.com/tinode/chat/commit/417b13f02fbf217d2ae0b1f4fe3e1f060e16c258 should address the issue.

**Comment by or-else** [comment]
This was released in 0.22.9
